nitric/proto/deployments/v1/deployments.proto

```diff
@@ -268,6 +268,20 @@ message WebsocketTarget {
   }
 }
 
+message Website {
+  // The production website output directory
+  string output_directory = 1;
+
+  // The index document for the website
+  string index_document = 2;
+
+  // The error document for the website
+  string error_document = 3;
+
+  // The base path for the website
+  string base_path = 4;
+}
+
```

Comment:
This looks good, the only thing that might be a bit of a sticking point is the ability to access these assets remotely if necessary. We have this issue in other areas of the protobuf spec as well but need to think about how this limits us in being able to host a nitric provider as a dedicated server vs. having to always run it locally. Might be worth including a `oneof` for the actual assets themselves so we have the option of including different asset type references in future.

Reply:
Perhaps something like:

```
// The source of the website assets
oneof asset_source {
  string local_directory = 5;
  string remote_url = 6;
}
```